servicemap/services.py

```diff
@@ -0,0 +1,43 @@
+from django.conf import settings
+
+from servicemap.rest_client import ServicemapApiClient
+
+service_map_api_client = ServicemapApiClient(config=settings.SERVICEMAP_API_CONFIG)
+
+
+class UnitService:
+    @staticmethod
+    def __normalize_service_unit_node(units: list):
```

Comment:
What is the need for the [name mangling](https://peps.python.org/pep-0008/#method-names-and-instance-variables) with the `__` prefix here? Could one leading underscore to mean non-public suffice in this case?

From https://peps.python.org/pep-0008/#method-names-and-instance-variables:
> Use one leading underscore only for non-public methods and instance variables.